src/unittest/helper_moveaction.lua

```diff
@@ -0,0 +1,19 @@
+core.debug = {}
+
+core.registered_allow_player_inventory_actions = {function(player, action, inventory, info)
+	if info.stack:get_name() == "default:water" then
+		return 0
+	end
+
+	if info.stack:get_name() == "default:lava" then
+		return 5
+	end
+
+	return info.stack:get_count()
+end}
+
+core.registered_on_player_inventory_actions = {function() end}
+
+core.run_callbacks = function(callbacks, _, ...)
```

Comment:
It might be feasible to import the real implementation.
Also for `core.debug` which I'm pretty sure needs to be a function.

Reply:
Fixed